gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1326,26 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+void
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env, MM_HeapWalkerObjectFunc walkFunction)
+{
+	HeapSizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
+
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	U_64 startTime = omrtime_hires_clock();
+
+	_heapWalker->allObjectsDo(env, walkFunction, &counter, MEMORY_TYPE_RAM, false, false, true);
+	/* While clearing actually requires 2 heap walks, we report it as one pass with cumulative time */
+	_extensions->globalGCStats.fixHeapForWalkTime += omrtime_hires_delta(startTime, omrtime_hires_clock(), OMRPORT_TIME_DELTA_IN_MICROSECONDS);
+	Assert_MM_true(FIXUP_NONE != _extensions->globalGCStats.fixHeapForWalkReason);
+	_extensions->globalGCStats.fixHeapForWalkReason = FIXUP_AND_CLEAR_HEAP;
```

Comment:
`_extensions->globalGCStats` should be stored in a local variable since it is accessed multiple times.